exercises/exercise_010_opaque_type_aliases_revised_stage_I/README.md

```diff
@@ -0,0 +1,76 @@
+# Opaque Type Aliases
+
+## Background
+
+An Opaque Type Alias can be used to provide the functionality of a "wrapper
+type" (i.e. a type that wraps, and therefore hides, another type) but without
+any runtime overhead. The aim is to provide additional type-safety at
+compile-time but then be stripped away at runtime. It is a powerful new feature
+of Scala 3 for supporting Information Hiding.
+
+Opaque Type Aliases differ from plain Scala 2 Type Aliases in that the later
+just provide a new name for a type but wherever this new name is used, the
+call-site still knows the details of the original type being aliased. With
+Opaque Type Aliases, the original type being aliased is hidden (or is opaque) at
+the call-site.
+
+## Steps
+
+We will change two type aliases, `ReductionSet` and `Sudoku` into opaque type
+aliases and we will do this in two steps tackling the conversion of
+`ReductionSet` first. Each time you change the code, recompile the project to see
+the effect. Don't forget to compile the tests too...
+
+> __Tip:__ Fix all of the compilation errors of this form `value {name} is not a
+      member of ...` before tackling the other types of error like `Found: ...
+      Required: ...`
+
+> __Tip:__ Some of the mission members for our new opaque type are generic (i.e.
```

Comment:
Typo: "mission" -> "missing"

Reply:
And fixed in [3e9da65](https://github.com/lunatech-labs/lunatech-scala-2-to-scala3-course/pull/85/commits/3e9da651f28ab83c374839a769a08ea4c15f2f34)